src/frontends/tensorflow_common/src/op/matrix_inverse.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/inverse.hpp"
+
+#include "common_op_table.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_matrix_inverse_op(const NodeContext& node) {
+    /*
+    For matrix inverse translation, the goal is to convert the TensorFlow input tensor
+    into an OpenVINO OutputVector that the Inverse operation can accept.
+    */
+    // a function that performs standard or default checks on a specific operation within the OpenVINO pipeline
+    default_op_checks(node, 1, {"MatrixInverse"}); // copied from issue #22964
+    auto op_type = node.get_op_type();
+    TENSORFLOW_OP_VALIDATION(node, op_type == "MatrixInverse", "Internal error: incorrect usage of translate_matrix_inverse_op.");
```

Comment:
```suggestion
```

Reply:
not needed because `default_op_checks` already performs this

Reply:
Oh, I see. Therefore we don't need to get the op_type. It will be repetitive for the code auto op_type=..